.github/workflows/build-plugin-zip.yml

```
@@ -93,10 +93,10 @@ jobs:
                       fi
                     fi
                   fi
-                  echo "::set-output name=new_version::$(echo $NEW_VERSION)"
+                  echo "new_versio=$(echo $NEW_VERSION)" >> $GITHUB_OUTPUT
```

Typo and likely no need for double echo.

```suggestion
                  echo "new_version=${NEW_VERSION}" >> $GITHUB_OUTPUT
```